src/js/libcs/OpDrawing.js

```diff
@@ -303,7 +303,7 @@ eval_helper.drawcircle = function(args, modifs, df) {
         [-xx, -yy, xx * xx + yy * yy - rad * rad]
     ]);
 
-    eval_helper.drawconic(cMat, modifs);
+    eval_helper.drawconic(cMat, modifs, df);
     */
```

Comment:
We might want to delete this whole comment, particularly seeing how the statement “drawconic doesn't handle filling” is no longer true. The comment dates from a5d0f6796095d8571d671e91ab208bc0c45016e3 but the commented-out code was superseded by #262 and later on by a fix for [Chrome issue #587582](https://bugs.chromium.org/p/chromium/issues/detail?id=587582).